packages/calypso-e2e/src/lib/flows/difm-flow.ts

```diff
@@ -0,0 +1,115 @@
+import { Page } from 'playwright';
+import { DataHelper } from '../..';
+
+const selectors = {
+	// General
+	button: ( text: string ) => `button:text("${ text }")`,
+
+	// Start page
+	startContainer: '.signup.is-do-it-for-me',
+
+	// Options page
+	siteNameInput: '#siteTitle',
+	taglineInput: '#tagline',
+
+	// Social page
+	facebookInput: 'input[name=facebookUrl]',
+
+	// Design page
+	designPickerContainer: '.design-picker',
+
+	//Page picker page
+	pagePickerContainer: '.signup__step .is-difm-page-picker',
+
+	// Checkout page
+	checkoutContainer: '.checkout__content',
+};
+
+/**
+ * Class encapsulating the flow when starting a new do-it-for-me site order.
+ */
+export class DIFMFlow {
+	/**
+	 * Constructs an instance of the flow.
+	 *
+	 * @param {Page} page The underlying page.
+	 */
+	constructor( private page: Page ) {}
+
+	/**
+	 * Given text, click on the button's first instance with the text.
+	 *
+	 * @param {string} text User-visible text on the button.
+	 */
+	async clickButton( text: string ): Promise< void > {
+		const selector = selectors.button( text );
+		await this.page.click( selector );
+	}
+
+	/**
+	 * Go to first setup page.
+	 */
+	async startSetup(): Promise< void > {
```

Comment:
FYI, other POM classes implement the same thing with the name `visit`.

Reply:
Ah thanks for pointing this out to me! 🙇🏻